### DISCUSSION TITLE
NextCloudPI LXC not installing

### DISCUSSION THREAD

**Discussion by nodeam** [tteck/Proxmox#4028] [Help Needed]
`Processing triggers for libc-bin (2.36-9+deb12u8) ...
 /curl: (6) Could not resolve host: raw.githubusercontent.com
/dev/fd/63: line 1: $'\E[?25h': command not found
/dev/fd/63: line 2: $'\E[01': command not found
/dev/fd/63: line 2: $'31m[ERROR]\E[m': command not found
/dev/fd/63: line 2: $'31m31\E[m:': command not found
/dev/fd/63: line 2: $'31m0\E[m:': command not found

[ERROR] in line 31: exit code 0: while executing command $STD bash <(curl -fsSL https://raw.githubusercontent.com/nextcloud/nextcloudpi/master/install.sh)`

**Comment by MickLesk** [comment]
Please create an issue at the new project, i will have take an look tomorrow.

https://github.com/community-scripts/ProxmoxVE
